tests/suites/test_suite_aes.cbc.data

```diff
@@ -71,145 +71,193 @@ AES-128-CBC Decrypt NIST KAT #12
 aes_decrypt_cbc:"00000000000000000000000000000000":"00000000000000000000000000000000":"39bde67d5c8ed8a8b1c37eb8fa9f5ac0":"fffffffffffffffffffffffffffffffc":0
 
 AES-192-CBC Encrypt NIST KAT #1
+depends_on:!MBEDTLS_AES_ONLY_128_BIT_KEY_LENGTH
```

Comment:
I have questions about the logic here. Is it better to mark all non-AES-128 tests as depending on `!MBEDTLS_AES_ONLY_128_BIT_KEY_LENGTH` or is it better to mark the AES-128 tests as depending on `MBEDTLS_CTR_DRBG_C` and `MBEDTLS_CTR_DRBG_USE_128_BIT_KEY`?

I am not sure of how the logic works with potentially conflicting config options. The documentation in `mbedtls_config.h` doesn't seem to explain it, unless I have missed it, and `check_config.h` doesn't have any rules for this.

Reply:
I think it's better to mark all non-AES-128 tests (AES-192 / AES-256) to depend on `!MBEDTLS_AES_ONLY_128_BIT_KEY_LENGTH` as this is the logic we design in `library/aes.c`. It's reasonable to keep same logic in test cases.

Assume `MBEDTLS_CTR_DRBG_C` is enabled. If we enable `MBEDTLS_AES_ONLY_128_BIT_KEY_LENGTH` then `MBEDTLS_CTR_DRBG_USE_128_BIT_KEY` should be enabled otherwise we got failure in `psa_crypto_init` because `mbedtls_ctr_drbg_seed` would set a `256-bit` key by default. That's why I added a rule in af07454

I didn't consider the case of `MBEDTLS_CTR_DRBG_C` disabled and `MBEDTLS_AES_ONLY_128_BIT_KEY_LENGTH` enabled before. (Should be modified later).

Reply:
Agreed,  AES tests that aren't 128 bit should depend on `!MBEDTLS_AES_ONLY_128_BIT_KEY_LENGTH`, not `xxx_CTR_DRBG_yyy`